### ISSUE TITLE
Backticks breaks the template editor

### ISSUE THREAD

**Issue by jimleroyer** [cds-snc/notification-planning#446]
## Describe the bug 

Putting in backticks in the template editor breaks it. The following characters insertion after a backtick produce weird results of characters inserted before or mixed with the new character.

![backticks.gif](https://images.zenhubusercontent.com/614b3e364039e00aa3137935/530c728b-5413-479b-8d27-6f6b857766cc)

Triple backticks also produce a different rendering on the email and would break the email parser:

![image.png](https://images.zenhubusercontent.com/614b3e364039e00aa3137935/948a8545-97d0-4afd-91df-16246bf7adbd)

## To Reproduce

1. Go to the template editor
2. Insert a backtick
3. Continue typing and moving around with the keyboard cursor.

## Expected behavior

Backticks act like in normal text editor and does not mess up with the text rendering when an email is sent.

## Impact

The impact would be low as few users use backticks, but for the ones who do, that would lead to garbled message that will confuse recipients. This could go under the radar for Notify users that produced such templates too.

## Additional context

@maxneuvians reported the bug [in these slack messages](https://gcdigital.slack.com/archives/CV38DBNVA/p1646342744277189).

**Comment by yaelberger-commits** [comment]
Need acceptance criteria that defines what "acting normal" looks like

**Comment by jimleroyer** [comment]
@andrewleith No, as far as I know. @maxneuvians reported it so we could contact him for further details on how to reproduce.

**Comment by schancase** [comment]
Confirming in team refinement that this is still relevant. Moving to product backlog.
